### ISSUE TITLE
Run BA after the VO example code. 

### ISSUE THREAD

**Issue by antithing** [openMVG/openMVG#2247]
Hi, i am looking into the example VO code (thanks!) But it seems to have some missing pieces, it outputs a sfm.bin file, but it does not seem to handle triangulation of 3d points, so no 3d landmarks nor camera poses are stored. Is this the expected result?

Thanks!

**Comment by pmoulon** [comment]
Yes this started as a prototype and I did not got time to work on it more.
Since that for VO the best is either to do BA for relative pose to global pose, or do full BA in a small window of poses and images with measurements.
My plan was to extend a per pose FIX constraint, so we could adjust a window of images, by having some images kept as fixed to avoid the entire scene to drift, but time get missing.

Let me know if my answer make sense to you
